admin_manual/configuration/user/user_provisioning_api.rst

```diff
@@ -245,6 +245,100 @@ XML Output
     <data/>
   </ocs>
 
+Enable User
+-----------
+
+Enable a user.
+Authentication is done by sending a Basic HTTP Authorization header.
+
+========================================== ======== ==============
+Request Path                               Method   Content Type
+========================================== ======== ==============
+``ocs/v1.php/cloud/users/{userid}/enable`` ``POST`` ``text/plain``
+========================================== ======== ==============
+
+======== ====== ======================================
+Argument Type   Description
+======== ====== ======================================
+userid   string The id of the user to enable
+======== ====== ======================================
+
+Status Codes
+^^^^^^^^^^^^
+
+* 100 - successful
+* 101 - failure
+
+Example
+^^^^^^^
+
+.. code-block:: console
+
+  # Enable the user ``Frank``
+  curl -X PUT http://admin:secret@example.com/ocs/v1.php/cloud/users/Frank/enable
+
+XML Output
+^^^^^^^^^^
+
+.. code-block:: xml
+
+  <?xml version="1.0"?>
+  <ocs>
+    <meta>
+      <status>ok</status>
+      <statuscode>100</statuscode>
+       <message/>
+    </meta>
+    <data/>
+  </ocs>
+
+Disable User
+-----------
+
+Disable a user.
```

Comment:
As above, this line is somewhat superfluous. Can you remove it in favour of a slightly meatier, yet still succinct, description?